server/util/prefix/prefix.go

```diff
@@ -19,26 +20,24 @@ func addPrefix(prefix, key string) string {
 }
 
 func userPrefixCacheKey(ctx context.Context, env environment.Env, key string) (string, error) {
-	if auth := env.GetAuthenticator(); auth != nil {
```

Comment:
note: the nil check isn't needed because we use `nullauth` instead of having a `nil` authenticator. If this were ever `nil`, then line 31 would have panicked in the current implementation.

Reply:
Wanna put this in a comment?

Reply:
Done